torch/fx/symbolic_trace.py

```diff
@@ -188,6 +188,38 @@ def forward(*args, **kwargs):
             torch.nn.Module.__call__ = orig_call
         return self.graph
 
+def wrap(orig_function):
+    """
+    A decorator that can be put on existing functions to use them in fx.
+
+        @torch.fx.wrap
+        def my_custom_function(x, y):
+            return x * x + y * y
+
+        sqrt = torch.fx.wrap(math.sqrt)
+
+    When called during symbolic tracing, the function will be inserted in the
+    graph rather than tracing it. When called outside of tracing,
+    the function will execute normally.
+    """
+
+    def wrapped(*args, **kwargs):
+        proxy = None
+
+        def find_proxy(x):
+            nonlocal proxy
+            if isinstance(x, Proxy):
+                proxy = x
+
+        map_aggregate(args, find_proxy)
+        map_aggregate(kwargs, find_proxy)
```

Comment:
I'm worried that it's pretty expensive to look for proxy every time when tracing is not on (what if there's a huge list passed around). Maybe just keep it an empty trampoline and replace the trampoline during tracing like we do for Module.__call__? (we could also use torch.fx.is_tracing, but that's not yet in main FX, right?)

even crazier idea is to make @wrap just keep name of the qualname and then monkeypatch it for the duration of tracing but it's a bit insane and doesn't work well with other decorators